generators/client/templates/angular/webpack/webpack.common.js.ejs

```diff
@@ -78,7 +78,16 @@ module.exports = (options) => ({
             'process.env': {
                 NODE_ENV: `'${options.env}'`,
                 BUILD_TIMESTAMP: `'${new Date().getTime()}'`,
-                VERSION: `'${utils.parseVersion()}'`,
+<%_ if (buildTool === 'maven') { _%>
+                // If webpack is invoked by Maven 'frontend-maven-plugin', then APP_VERSION is set.
+                VERSION: `'${process.env.hasOwnProperty('APP_VERSION') ? process.env.APP_VERSION : 'UNKNOWN'}'`,
+<%_ } else if (buildTool === 'gradle') { _%>
```

Comment:
can we combine `if` and `else if`?

Reply:
I am not sure if this makes it easier to read (since the comments above are different). What do you think?

Reply:
Maybe we can make the comment `If webpack is invoked via Maven/Gradle then APP_VERSION is set` as the code is the same?

Reply:
That seems to be a more pragmatic solution :). Thanks. Will change.